classes/decorate.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'Nameable'
+
+class BaseDecorator < nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @correct_name.correct_name
+  end
+end
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0, 10] if @nameable.correct_name.length > 10
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Generally it is best practice to keep your classes in separate files that include the class name in the file name. This keeps your project better organized and makes it easier for someone else to navigate your structure. Please fix it :smile:

Reply:
@codecaiine  
Thanks for your perfect review and wonderful feedback